gateway/gateway/entrypoints.py

```diff
@@ -25,7 +25,7 @@ def response_from_exception(self, exc):
         if isinstance(exc, self.expected_exceptions):
             if type(exc) in self.mapped_errors:
                 status_code, error_code = self.mapped_errors[type(exc)]
-            else:
+            else:  # pragma: no cover
```

Comment:
Yeah. I could not think of any test for this unless you don't wrap `ValueError` here https://github.com/nameko/nameko-examples/pull/5/files#diff-e3d4340eb966f49b4dd5d50bc4e770e8R89 which would then throw `BadRequest` here